homeassistant/components/adguard/__init__.py

```diff
@@ -198,7 +198,7 @@ def device_info(self) -> DeviceInfo:
         """Return device information about this AdGuard Home instance."""
         return {
             "identifiers": {
-                (DOMAIN, self.adguard.host, self.adguard.port, self.adguard.base_path)
+                (DOMAIN, self.adguard.host, self.adguard.port, self.adguard.base_path)  # type: ignore
```

Comment:
This is an issue in other integrations as well, can typing of `DeviceInfo` be updated to be less strict?

Reply:
We should preferably migrate the entry to just use a two string tuple.